posts/2022-06-07-microprofile-graphql-2-22006.adoc

```diff
@@ -77,6 +78,123 @@ To enable the MicroProfile GraphQL 2.0 feature, add it to your `server.xml`:
 </featureManager>
 ----
 
+[#db2]
+===  Improved DB2 Query Performance
+This new JPA enhancement improves EclipseLink's DB2 and DB2 z/OS support by enabling parameter binding for JPA specification defined functions.
+
+For DB2 and DB2 for z/OS, EclipseLink's JPQL parameter binding is disabled when function expressions (SQRT, ABS, SUBSTR, COALESCE, etc.) exist within a JPQL query. This feature will allow EclipseLink to bind JPQL parameters for function expressions that accept untyped parameter markers on DB2 and DB2 z/OS. 
+This feature will also allow EclipseLink to enable/disable parameter binding per expression, within a query, rather then for the entirety of the query.
+
+For DB2 and DB2 for z/OS, EclipseLink does not bind JPQL literal values as SQL parameter values. An EclipseLink specific persistence property currently exists `("shouldBindLiterals=true")` to allow EclipseLink users to bind all literal values as parameter values. For DB2 and DB2 for z/OS, this property is currently ignored by EclipseLink.
+This feature will allow for the usage of the `"shouldBindLiterals=true"` EclipseLink persistence property to bind literals values as parameters on DB2 and DB2 z/OS; assuming it is legal to do so according to the database.
+
+==== Enabling the New Functionality
+In order to start using the new feature, users need only add the new persistence property `eclipselink.jdbc.allow-partial-bind-parameters` to their `persistence.xml` properties:
+
+[source, xml]
+----
+<persistence ... >
+    <persistence-unit ... >
+        <properties>
+            <property name="eclipselink.jdbc.allow-partial-bind-parameters" value="true"/>
+        </properties>
+     </persistence-unit>
+</persistence>
+----
+
+Alternatively, starting on Liberty 22.0.0.2 releases, users need only add the new `<defaultProperties>` configuration to their `<jpa>` configuration, in server.xml. This will apply the persistence property `eclipselink.jdbc.allow-partial-bind-parameters` to all persistence units within applications running on this server.  For example:
```

Comment:
this is confusing- is the beta update only to add the property in a persistence.xml? It seems to be saying the capability has existed for the server.xml since 22.0.0.2. If the capability is only currently available in the beta, do we need to mention 22.0.0.2?